### PR TITLE
Fix bug: range indexlabel check not correct

### PR DESCRIPTION
**Pull request by Linary**
#1009

Change-Id: I8f28f46cc1a0a0edd88f381863bacc78d0408d4e

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java [hunk 1]
```diff
@@ -125,9 +125,9 @@ public IndexLabel build() {
     private boolean hasSameProperties(IndexLabel existedIndexLabel) {
         // baseType is null, it means HugeType.SYS_SCHEMA
         if ((this.baseType == null &&
-             existedIndexLabel.baseType() != HugeType.SYS_SCHEMA) ||
-            (this.baseType != null &&
-             this.baseType != existedIndexLabel.baseType())) {
+                     existedIndexLabel.baseType() != HugeType.SYS_SCHEMA) ||
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java:128]: align

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java [hunk 2]
```diff
@@ -125,9 +125,9 @@ public IndexLabel build() {
     private boolean hasSameProperties(IndexLabel existedIndexLabel) {
         // baseType is null, it means HugeType.SYS_SCHEMA
         if ((this.baseType == null &&
-             existedIndexLabel.baseType() != HugeType.SYS_SCHEMA) ||
+            existedIndexLabel.baseType() != HugeType.SYS_SCHEMA) ||
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java:128]: add one space before existedIndexLabel

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java [hunk 3]
```diff
@@ -125,9 +125,9 @@ public IndexLabel build() {
...
             (this.baseType != null &&
-             this.baseType != existedIndexLabel.baseType())) {
+            this.baseType != existedIndexLabel.baseType())) {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/IndexLabelBuilder.java:130]: ditto

**Comment by zhoney** [comment]
```
[ERROR] Tests run: 673, Failures: 0, Errors: 1, Skipped: 16, Time elapsed: 485.711 s <<< FAILURE! - in com.baidu.hugegraph.core.CoreTestSuite
[ERROR] testRemoveEdgesOfSuperVertex(com.baidu.hugegraph.core.EdgeCoreTest)  Time elapsed: 5.052 s  <<< ERROR!
java.lang.IllegalArgumentException: The server with name 'server1' already in cluster
	at com.baidu.hugegraph.core.EdgeCoreTest.testRemoveEdgesOfSuperVertex(EdgeCoreTest.java:5310)
2020-07-14 11:48:58 488595 [Thread-2] [INFO ] com.baidu.hugegraph.HugeGraph [] - HugeGraph is shutting down
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   EdgeCoreTest.testRemoveEdgesOfSuperVertex:5310 » IllegalArgument The server wi...
```

**Pull request merged**